src/uu/ls/src/ls.rs

```diff
@@ -669,8 +753,25 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 .require_equals(true)
                 .min_values(0),
         )
+        .arg(
+            Arg::with_name(options::INDICATOR_STYLE)
+                .long(options::INDICATOR_STYLE)
+                .help(" append indicator with style WORD to entry names: none (default),  slash\
+                       (-p), file-type (--file-type), classify (-F)")
+                .takes_value(true)
+                .overrides_with_all(&[
+                    options::FILE_TYPE,
+                    options::SLASH,
+                    options::CLASSIFY,
+                    options::INDICATOR_STYLE,
+                    options::indicator_style::NONE,
+                    options::indicator_style::SLASH,
+                    options::indicator_style::FILE_TYPE,
+                    options::indicator_style::CLASSIFY,
+                ])
+        )
```

Comment:
Could you group the arguments, so they're all next to each other in the code?

Reply:
And maybe specify the possible values (with `Arg::possible_values`), so we get nice clap errors if an invalid value is given.

Reply:
Sure thing! Thanks!